web-registry/src/components/organisms/BuyCreditsModal/BuyCreditsModal.tsx

```diff
@@ -203,36 +210,53 @@ const BuyCreditsModal: React.FC<BuyCreditsModalProps> = ({
                             name="creditCount"
                             min={1}
                             max={selectedSellOrder?.quantity}
+                            validate={getCreditCountValidation(
+                              Number(selectedSellOrder?.quantity),
+                            )}
                           />
                         </div>
-                        <Title variant="h6" sx={{ mr: 4 }}>
-                          =
-                        </Title>
-                        <div
-                          className={cx(styles.flexColumn, styles.marginRight)}
+                        <Box
+                          sx={{
+                            display: 'flex',
+                            alignItems: 'center',
+                            height: 60,
+                          }}
```

Comment:
Add a wrapper with some style to keep the same alignment with and without an error message.